tools/pd-ctl/pdctl/ctl.go

```diff
@@ -31,23 +35,38 @@ type CommandFlags struct {
 
 var (
 	commandFlags = CommandFlags{}
+
+	detach   bool
+	interact bool
+	version  bool
 )
 
 func init() {
 	cobra.EnablePrefixMatching = true
 }
 
-// Start run Command
-func Start(args []string) {
+func pdctlRun(cmd *cobra.Command, args []string) {
+	if version {
+		server.PrintPDInfo()
+		return
+	}
+	if interact {
+		loop()
+	}
+}
+
+func getBasicCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:   "pd-ctl",
 		Short: "Placement Driver control",
 	}
+
 	rootCmd.PersistentFlags().StringVarP(&commandFlags.URL, "pd", "u", "http://127.0.0.1:2379", "pd address")
```

Comment:
It's better to change all the first letters to the upper case.

Reply:
Do you mean change all first letters of functions in ctl.go to upper case?

Reply:
oh, sorry for misleading. What I mean is the help message.

Reply:
In fact, only the MainStart() needs to export to package main.

Reply:
> It's better to change all the first letters to the upper case.

OK, change the first letter in message to upper case now.